internal/bminventory/inventory.go

```diff
@@ -1884,6 +1884,41 @@ func (b *bareMetalInventory) updateDhcpNetworkParams(updates map[string]interfac
 	return nil
 }
 
+func (b *bareMetalInventory) updateKubeAPINetworkParams(
```

Comment:
Why is there a specific code for kube api? 
The ticket talked about removing `MachineNetworkCidr` from REST API as well.
It will be validated in async validation.